### ISSUE TITLE
Wordpress dashboard PHP fatal error

### ISSUE THREAD

**Issue by marklenahan** [Jonnyauk/Wonderflux#53]
Full text of the error from the server error_log is:-
[Sun Feb 28 00:18:26 2016] [error] PHP Fatal error:  Call to a member function filter_post_class() on null in /var/www/vhosts/46/463094/webspace/siteapps/WordPress-97267/htdocs/wp-content/themes/wonderflux/functions.php on line 764

This happens only when I'm in the Wordpress dashboard, nothing seems to appear while using the site itself.

I honestly can't find what, if any, usability or functionality issue this bug causes. I'm having a problem with my feeds, in that the RSS2 or atom XML are stopping at the site image and not actually listing posts (or even closing the rss or channel tags). But I don't know if this error relates to that.
You can see the incomplete feed here:-
http://marklenahan.com/feed/

Technical and server information:-
Wonderflux version installed: 2.1, requires at-least WordPress v4.0, optimised for WordPress v4.4
WordPress version installed: 4.4.2
PHP version installed: 5.6.18
MySQL version installed: 5.5.48

**Comment by Jonnyauk** [comment]
Hi Mark,
thanks for the bug report - I'm having trouble replicating that issue (related to filter_post_class()), but do have an idea on what might be causing it - that was pushed out as a fairly late amend just before the last release, so I've a good idea on how to sort that out and get it fixed ;)

Regarding your feeds - humph - that's a strange one - are you running any other plugins at-all?

**Comment by marklenahan** [comment]
Hi Johnny,

Thanks for taking this one. 

I resolved (or worked around) the rss/atom feed issue - I had an ampersand in a blog title, once I changed the title to use the word "and" the rss2 and atom XMLs started working again. The truncation was happening _whether or not_ Wonderflux was the active theme.

In any case, back to your functions.php, once I updated the title of the post, the PHP error also _stopped_ appearing in the logs. 

So maybe this is not in Wonderflux at all? (Or might be a non-issue).

Maybe putting an ampersand in the blog title is the way to reproduce the issue? If not, and you still want to look at this, are there any other details I can gather? I could embed some log code in the function if you want? Right now this is not impacting me in any way I can see but I'm more than happy to help if this is still a topic for concern.

Concerning the potential Wordpress issue I'm not sure if using an ampersand in blog titles is just not allowed (I can see why it is probably a bad idea), but I raised an issue in wordpress support about it anyway:-
https://wordpress.org/support/topic/feed-xml-rss-or-atom-truncated-no-posts-appearing?replies=2

My plugins at the moment are (all latest versions):-
Google Analytics Dashboard for WP, Version 4.9.2 | By Alin Marcu 
Jetpack by WordPress.com, Version 3.9.2 | By Automattic 
TablePress, Version 1.7 | By Tobias Bäthge 
WP Editor Widget, Version 0.5.5 | By David Mårtensson

I tried disabling all of them one by one, while working on the feed issue and the PHP error_log, but none of them seemed to effect it.

Thanks, Mark
